wofs/wofs_app.py

```diff
@@ -456,30 +467,45 @@ def estimate_job_size(num_tasks):
 @click.group(help='Datacube WOfS')
 @click.version_option(version=__version__)
 def cli():
+    """
+    Instantiate a click.group object
+    :return: None
+    """
     pass
 
 
 @cli.command(name='list', help='List installed WOfS config files')
 def list_configs():
-    for cfg in CONFIG_DIR.glob('*.yaml'):
+    """
+     List installed WOfS config files
+    :return: None
+    """
+    for cfg in (glob.glob(os.path.abspath(os.path.join(WOFS_CONFIG_DIR,
+                                                       "wofs/config/*.yaml")),
+                          recursive=True)):
         click.echo(cfg)
 
 
 @cli.command(
     help="Ensure the products exist for the given WOfS config, creating them if necessary."
 )
 @task_app.app_config_option
+@click.option('--dry-run', is_flag=True, default=False, help='Check if the product definition exist in db')
 @ui.config_option
 @ui.verbose_option
 @ui.pass_index(app_name=APP_NAME)
-def ensure_products(index, app_config_files):
-    for app_config_file in app_config_files:
+def ensure_products(index, app_config, dry_run):
+    """
+    Ensure the products exist for the given WOfS config, creating them if necessary.
+    If dry run is enabled, the validated output product definition will be added to the database.
+    """
+    for app_config_file in [app_config]:
```

Comment:
This `for` loop should be removed, it does nothing.

Reply:
`for` loop removed